docs/source-fabric/index.rst

```diff
@@ -60,20 +60,20 @@ Fabric is the fast and lightweight way to scale PyTorch models without boilerpla
 Why Fabric?
 ***********
 
-Fabric differentiates itself from a fully-fledged trainer like :doc:`Lightning Trainer <../common/trainer>` in these key aspects:
+Fabric differentiates itself from a fully-fledged trainer like Lightning Trainer in these key aspects:
```

Comment:
Can't we have links between top level projects?

Reply:
Not as they are generated as very independent documentation; we may later explore https://docs.readthedocs.io/en/stable/subprojects.html (which we tried already in past/a few months ago, but did not work well)

Reply:
```suggestion
Fabric differentiates itself from a fully-fledged trainer like Lightning :class:`pytorch_lightning.Trainer` in these key aspects:
```